rohmu/encryptor.py

```diff
@@ -208,23 +244,23 @@ def _reset(self):
         self.state = "OPEN"
 
     @classmethod
-    def _file_size(cls, file):
+    def _file_size(cls, file: io.BufferedIOBase) -> int:
```

Comment:
nit: this is really weird classmethod and not your problem but it could be really something outside class too.